src/app/tech-carbon-estimator/tech-carbon-estimator.component.html

```diff
@@ -11,7 +11,7 @@ <h1 class="tce-text-3xl tce-mb-6">Technology Carbon Estimator</h1>
             (formReset)="handleFormReset()"></carbon-estimator-form>
         </tab-item>
         <tab-item [title]="'Assumptions and Limitations'">
-          <assumptions-and-limitation aria-live="polite"></assumptions-and-limitation>
```

Comment:
I originally removed this when changing some other aria-live values but having reverted those changes and rechecked, I don't think this is changing the behaviour although that could just be my screen reader settings. I've left it off for now since other elements placed directly inside the tab-item components do not have this set.